src/haddock/modules/analysis/clustfcc/defaults.yaml

```diff
@@ -1,51 +1,51 @@
 executable:
   default: src/contact_fcc
   type: file
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
-  explevel: easy
+  title: Relative path to contact_fcc executable.
+  short: Relative path to contact_fcc executable.
+  long: As contact_fcc is an external tool, its path must be provided. Basically, this parameter should not be changed, as the tool is downloaded at the same time as haddock3. Nevertheless, one could try to have the contact_fcc tool written in another programming language or whatever.
+  group: executable
+  explevel: expert
 contact_distance_cutoff:
   default: 5.0
   type: float
-  min: -9999
-  max: 9999
+  min: 0.000
+  max: 999.999
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Cutoff distance (A) for contact detection.
+  short: Cutoff distance (A) for contact detection. By default 5.0 Angstroms.
+  long: Cutoff distance between two atoms for contact detection. By default 5.0 Angstroms.
+  group: analysis
   explevel: easy
 fraction_cutoff:
-  default: 0.6
+  default: 0.600
   type: float
-  min: -9999
-  max: 9999
+  min: 0.000
+  max: 1.000
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Fraction of common contacts to not be considered a singleton model.
+  short: Fraction of common contacts to not be considered a singleton model and therefore be excluded by further clustering. By default 0.6.
+  long: Minium fraction of common contacts (fcc) to cluster complexes together. Complexes not sharing fraction of common contacts above the `fraction_cutoff` parameter cannot belong to the same cluster.
+  group: analysis
   explevel: easy
 threshold:
   default: 4
   type: integer
-  min: -9999
+  min: 1
   max: 9999
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Clustering population threshold.
+  short: Threshold employed to exclude clusters with less than this number of members. By default 4.
+  long: Minimum set of grouped complexes to define a cluster. If a potential cluster is composed of less than this number of members, it will be considered as `singloton` and discarded from final clusters. If set to 1, all complexes will be part of a cluster. By default 4.
+  group: analysis
   explevel: easy
 strictness:
```

Comment:
Isn't this parameter redundant with the fraction_cutoff ? I.e. can be removed (should make sure it is no longer used) Now we have inconsistent values (0.6 vs 0.75)

Reply:
I think those two parameters address two different tasks..the fraction_cutoff tells you if a model will belong to any cluster (if it has fcc < 0.6 wrt all the other models then it's a "singleton"), while the strictness controls when you should put complexes in the same cluster..I am not 100% sure though

Reply:
The haddock2.4 default for FCC clustering is 0.6.  Which is the default for the fraction_cutoff.

I don't see why we should have two different parameters for those two tasks. The same parameter can be used for both tasks, with > to make it cluster and < to make it a singleton